tools/pipelines/test-real-service.yml

```diff
@@ -93,9 +93,6 @@ stages:
             flags: --compatVersion=LTS
           - name: Cross-version
             flags: --compatVersion=CROSS_VERSION
-          # Assumes Non-compat and N-1 scenarios are covered
```

Comment:
How do we know these scenarios are not supported for AFR / routerlicious?

Reply:
We don't officially support this, we support only AFR scenarios